encord/workflow/workflow.py

```diff
@@ -77,6 +81,17 @@ def get_stage(
         uuid: Optional[UUID | str] = None,
         type_: Optional[Type[WorkflowStageT]] = None,
     ) -> WorkflowStageT:
+        """
+        **Params**
+
+        - name: Name of the stage.
+        - uuid: Unique identifier for the stage.
+        - type_: The type of stage.
+
+        **Returns**
+
+        Returns Workflow stages from non-Consensus and Consensus Projects.
```

Comment:
Possibly worth commenting on the stage being coerced to be of type type_

Reply:
Updated